### DISCUSSION TITLE
Friedrich Nietzsche was a what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#142] [Galaxy Trivia] [answered]
**Hint:** 🧠🏥⚕️

**Comment by Tynab** [comment, marked as answer]
Friedrich Nietzsche was a German philosopher, cultural critic, and poet. He lived from 1844 to 1900 and is widely considered one of the most influential philosophers of the modern era. Nietzsche's work focuses on themes such as morality, religion, culture, and the nature of human existence, and he is particularly known for his critiques of traditional values and beliefs. Some of his most famous works include "Thus Spoke Zarathustra," "Beyond Good and Evil," and "On the Genealogy of Morals."
